### PR TITLE
Remove legacy vendoring

### PR DESCRIPTION
**Pull request by wojpawlik**
It's complex and unusably slow.

Vendoring is still marked experimental, so this shouldn't require bumping major version. Still, to provide a grace period, this shouldn't be merged before 1.2.0.

### CODE REVIEW

File: src/_transformations/vendor.ts [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 import type { Context } from "../context.ts";
-import * as cache from "../cache.ts";
 
 interface LocalFile {
   readonly sourceFile: SourceFile;
   readonly url: URL;
 }
 
 function getVendoredPath(url: URL) {
+  // FIXME respect ${vendorDir}/import_map.json
```
Comment (KnorpelSenf) [inline src/_transformations/vendor.ts:10]: I guess this should be fixed before merging?

Otherwise, I very much approve of this!

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: src/_transformations/vendor.ts (hunk 1 above)
Comment (wojpawlik) [reply src/_transformations/vendor.ts:10]: Import map support is non-trivial. It deserves a separate PR.

Comment (wojpawlik) [reply src/_transformations/vendor.ts:10]: To correctly resolve relative imports within vendored modules, we need to map them back to remote URLs; I'm not sure how to do that yet

**Comment by wojpawlik** [comment]
1.2.0 failed to build and release because of a race condition in legacy vendoring... clean break it is then.

**Pull request merged**